easybuild/easyblocks/h/healpix.py

```diff
@@ -112,20 +143,27 @@ def install_step(self):
         """No dedicated install procedure for HEALPix."""
         pass
 
+    def make_module_extra(self):
+        """additional paths"""
+        txt = super(EB_HEALPix, self).make_module_extra()
+        txt += self.module_generator.prepend_paths('PATH', os.path.join('src/cxx', self.target_string, 'bin'))
+        txt += self.module_generator.prepend_paths('LIBRARY_PATH', os.path.join('src/cxx', self.target_string, 'lib'))
+        txt += self.module_generator.prepend_paths('CPATH', os.path.join('src/cxx', self.target_string, 'include'))
+        return txt
+
     def sanity_check_step(self):
-        """Custom sanity check for HEALPix < 3.50"""
-        # since 3.50 files changed, so we specify it in easyconfig.
-        # "else" section below is for backward-compatibility, ATM only easyconfig for 2.20a-icte is known
-        if LooseVersion(self.version) >= LooseVersion('3.50'):
-            super(EB_HEALPix, self).sanity_check_step()
-        else:
-            custom_paths = {
-                'files': [os.path.join('bin', x) for x in ['alteralm', 'anafast', 'hotspot', 'map2gif', 'median_filter',
-                                                           'plmgen', 'sky_ng_sim', 'sky_ng_sim_bin', 'smoothing',
-                                                           'synfast', 'ud_grade']] +
-                         [os.path.join('lib', 'lib%s.a' % x) for x in ['chealpix', 'gif', 'healpix', 'hpxgif',
-                                                                       'psht_healpix_f']] +
-                         [os.path.join('lib', 'libchealpix.%s' % get_shared_lib_ext())],
-                'dirs': [os.path.join('include')],
-            }
-            super(EB_HEALPix, self).sanity_check_step(custom_paths=custom_paths)
+        """sanity checks"""
+        custom_paths = {
+            'files': [os.path.join('bin', x) for x in ['alteralm', 'anafast', 'hotspot', 'map2gif', 'median_filter',
+                                                       'plmgen', 'sky_ng_sim', 'sky_ng_sim_bin', 'smoothing',
+                                                       'synfast', 'ud_grade']] +
+                     [os.path.join('lib', 'lib%s.a' % x) for x in ['chealpix', 'gif', 'healpix', 'hpxgif']] +
+                     [os.path.join('lib', 'libchealpix.%s' % get_shared_lib_ext())],
+            'dirs': [
+                os.path.join(self.installdir, 'include'),
+                os.path.join(self.installdir, 'src/cxx', self.target_string, 'bin'),
+                os.path.join(self.installdir, 'src/cxx', self.target_string, 'lib'),
+                os.path.join(self.installdir, 'src/cxx', self.target_string, 'include'),
```

Comment:
the `self.installdir` here is not actually needed (paths are checked relative to the installation directory)